.gitignore

```diff
@@ -21,3 +21,7 @@ obj/
 .vscode
 
 *.html
+
+# Eclipse project folder
+/.project
+/.externalToolBuilders
```

Comment:
Please move these in `.git/info/exclude` on your machine.

Reply:
will do